### DISCUSSION TITLE
Adding Airsim Plugin

### DISCUSSION THREAD

**Discussion by trmthcode3** [microsoft/AirSim#4391] [Support Q&A]
Hello all, so I am using this video https://www.youtube.com/watch?v=1oY8Qu5maQQ as guidance to add the AirSim plugin to landscape mountains. I can get all the way through all the steps, however whenever I try to open MyProject from the Unreal editor I get this error

"The following modules are missing or built with a different engine version: LandscapeMountains Would you like to rebuild them now?"

I opened the status file and see the only error looks like "LogInit: Warning: Incompatible or missing module: LandscapeMountains"

Anyone know what I need to do to solve this? When I click rebuild I get another error.

**Comment by jonyMarino** [comment]
#4392
